smart-contracts/contracts/BridgeBank/BridgeBank.sol

```diff
@@ -500,10 +516,6 @@ contract BridgeBank is BankStorage,
         // decimals defaults to 18 if call to decimals fails
         uint8 decimals = getDecimals(tokenAddress);
 
-        // Get name and symbol
-        string memory name = getName(tokenAddress);
-        string memory symbol = getSymbol(tokenAddress);
```

Comment:
We should call out to the bridgetoken here and get the cosmos denom and then modify the `LogBurn` message to emit the cosmos denom in the message.